### PR TITLE
Add PyChart 1.39 as pychart in the Python packages

### PR DESCRIPTION
**Pull request by tinkerthaler**
I'm rather new to NixOS and nixpkgs but would like to add some python packages such as this one: PyChart (latest version 1.39 from http://home.gna.org/pychart/)

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -12771,4 +12771,19 @@ let
...
+  pychart = buildPythonPackage rec {
+    name = "pychart-1.39";
+
+    src = pkgs.fetchurl {
+      url = "http://download.gna.org/pychart/PyChart-1.39.tar.gz";
+      md5 = "f1f509a1c4623056c8e780bb7c9a05c5";
+    };
+
+    meta = with stdenv.lib; {
+      description = "PyChart is a Python library for creating high quality Encapsulated Postscript, PDF, PNG, or SVG charts.";
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: The package name should not be containted in the description. See http://nixos.org/nixpkgs/manual/#idm140737319048304

**Comment by tinkerthaler** [comment]
Description updated.

**Comment by pSub** [comment]
Ah, please use a sha256 has instead of md5 and squash the commits. Otherwise the PR looks fine!

**Comment by tinkerthaler** [comment]
Not sure why the build failed on the commit. I'd be tempted to just close this PR and create a new one to trigger a new build.

If not I can look at it later but I don't understand the error (yet). Nor what I has to do with my change of adding pychart. 

I was able to ```nix-build -A dictdWiktionary``` locally. 

I was not able to run ```./maintainers/scripts/travis-nox-review-pr.sh build``` locally (I'd need to check my system configuration) nor ```nix-build pkgs/top-level/release.nix -A tarball``` (not sure if that _should_ work but I see: /nix/store/frmyf8fdq5xhm035mr87wn6kj0c278g7-stdenv/setup: line 851: 38017 Killed                  NIXPKGS_ALLOW_BROKEN=1 nix-env -f pkgs/tr \* --system --meta --xml > /dev/null)

The info from the Travis CI is:
```
error: while querying the derivation named ‘dict-db-wiktionary-20140118’:
while evaluating the attribute ‘args’ of the derivation ‘dict-db-wiktionary-20140118’ at "/tmp/nix-build-nixpkgs-tarball-15.05pre1234.abcdef.drv-0/nixpkgs/pkgs/servers/dict/dictd-wiktionary.nix":5:3:
while evaluating the attribute ‘text’ of the derivation ‘wiktionary-builder.sh’ at "/tmp/nix-build-nixpkgs-tarball-15.05pre1234.abcdef.drv-0/nixpkgs/pkgs/build-support/trivial-builders.nix":10:14:
cannot coerce null to a string, at "/tmp/nix-build-nixpkgs-tarball-15.05pre1234.abcdef.drv-0/nixpkgs/pkgs/servers/dict/dictd-wiktionary.nix":14:49
```

Some more details:

```
...
lrwxrwxrwx 1 travis travis 66 Jan 19 13:05 result -> /nix/store/0nnmyvj574dmqbw76311wqfx52zb2m82-python2.7-pychart-1.39

=== Checking tarball creation
these derivations will be built:
/nix/store/j4ld9vlnldrmwk8blw7a3snza8lpbfxb-nixpkgs-tarball-15.05pre1234.abcdef.drv
these paths will be fetched (4.00 MiB download, 16.07 MiB unpacked):
...<cut>...
building /nix/store/9yy7k0fxrqaj8bpcbzmb9a69a52qm41n-nixpkgs-tarball-15.05pre1234.abcdef
unpacking sources
unpacking source archive /nix/store/4c0bi12idc3mwmkayjsk7bbd28vn8gjs-nixpkgs
source root is nixpkgs
patching sources
autoconfPhase
No bootstrap, bootstrap.sh, configure.in or configure.ac. Assuming this is not an GNU Autotools package.
configuring
release name is nixpkgs-15.05pre1234.abcdef
building
no Makefile, doing nothing
running tests
warning: you did not specify ‘--add-root’; the result might be removed by the garbage collector
warning: you did not specify ‘--add-root’; the result might be removed by the garbage collector
checking pkgs/top-level/all-packages.nix on i686-linux
checking pkgs/top-level/all-packages.nix on x86_64-linux
checking pkgs/top-level/all-packages.nix on x86_64-darwin
error: while querying the derivation named ‘dict-db-wiktionary-20140118’:
while evaluating the attribute ‘args’ of the derivation ‘dict-db-wiktionary-20140118’ at "/tmp/nix-build-nixpkgs-tarball-15.05pre1234.abcdef.drv-0/nixpkgs/pkgs/servers/dict/dictd-wiktionary.nix":5:3:
while evaluating the attribute ‘text’ of the derivation ‘wiktionary-builder.sh’ at "/tmp/nix-build-nixpkgs-tarball-15.05pre1234.abcdef.drv-0/nixpkgs/pkgs/build-support/trivial-builders.nix":10:14:
cannot coerce null to a string, at "/tmp/nix-build-nixpkgs-tarball-15.05pre1234.abcdef.drv-0/nixpkgs/pkgs/servers/dict/dictd-wiktionary.nix":14:49
build time elapsed: 0m0.041s 0m0.041s 1m1.819s 0m2.538s
builder for ‘/nix/store/j4ld9vlnldrmwk8blw7a3snza8lpbfxb-nixpkgs-tarball-15.05pre1234.abcdef.drv’ failed with exit code 1
error: build of ‘/nix/store/j4ld9vlnldrmwk8blw7a3snza8lpbfxb-nixpkgs-tarball-15.05pre1234.abcdef.drv’ failed
The command "./maintainers/scripts/travis-nox-review-pr.sh build" exited with 100.
```
Done. Your build exited with 1.

**Pull request merged**